index.bs

```diff
@@ -1036,6 +1037,10 @@ An <dfn>attribution rate-limit record</dfn> is a [=struct=] with the following i
 :: A [=moment=].
 : <dfn>expiry time</dfn>
 :: Null or a [=moment=].
+: <dfn>entity ID</dfn>
```

Comment:
How do we expect this field to work in the context of Full Flex? There may not be a persistent identifier corresponding to a report at the time of attribution, since reports under that proposal are effectively computed at the end of each report window by a cumulative operation across all triggers received during the window.

I assume this whole notion of rate-limit-row replacement logic is going to have to be itself replaced as part of Full Flex?

Reply:
I don't thing this whole thing will need to replaced.

Whenever report re-computation happens, we can delete the obsolete rate-limit records and add new ones.

Reply:
I'm not sure the spec is going to be written using report re-computation, as that's more of an implementation detail, but we can deal with that when Full Flex is specified.

Reply:
Ack, we can revisit this when we implement Full Flex. Also, this field will be useful for #1217 regardless of Flex implementation.